sdks/python/tox.ini

```diff
@@ -144,6 +144,7 @@ deps =
   sphinx_rtd_theme==0.4.3
   docutils<0.18
   Jinja2==3.0.3 # TODO(BEAM-14172): Sphinx version is too old.
+  torch==1.9.0
```

Comment:
I don't think we should hardcode a particular version of torch, since Beam doesn't have a requirement on torch.